OpenRA.Mods.Common/Activities/Attack.cs

```diff
@@ -195,12 +195,10 @@ protected virtual AttackStatus TickAttack(Actor self, AttackFrontal attack)
 					return AttackStatus.UnableToAttack;
 
 				attackStatus |= AttackStatus.NeedsToMove;
-
 				var checkTarget = useLastVisibleTarget ? lastVisibleTarget : target;
-				moveActivity = ActivityUtils.SequenceActivities(
+				moveActivity = ActivityUtils.SequenceActivities(self,
 					move.MoveWithinRange(target, minRange, maxRange, checkTarget.CenterPosition, Color.Red),
 					this);
-
```

Comment:
Readability nit: I think at least this empty line should stay.

Reply:
Fixed.